src/policy/CMSRucioPolicy/permission.py

```diff
@@ -217,10 +329,21 @@ def perm_add_rule(issuer, kwargs, *, session: "Optional[Session]" = None):
     if all_temp and kwargs['lifetime'] is not None and kwargs['lifetime'] < 31 * 24 * 60 * 60:
         return True
 
+    # FIXME: This should be removed soon, we can move sync data to a single account, say phedex
```

Comment:
Didn't we have a public removal of all sync rules? I think we can assume no more are coming

Reply:
Okay, then I will remove the permissions. For moving the rules to a single account (we can do it using other accounts).